WTWSMS/common/trade_routes/01_amber_route.txt

```diff
@@ -0,0 +1,21 @@
+amber_route = {
+
+	wealth = 10			# Total base wealth of the amber road
+
+	modifier = {			# How the trade route modifies the provinces it passes
+		castle_tax_modifier = 0.02
+		city_tax_modifier = 0.05
+		temple_tax_modifier = 0.01
+		trade_route_value = 0.5
+		icon = 4
+	}
+	
+	start = {
+		414
+	}
+	
+	# Eastern roads
+	path = {	# Main Eastern Road  Novgorod-Riga-Danzig-Olomouc-Venice-Rome
```

Comment:
I would say Aquilea instead of Venice, as we discussed.

Reply:
Right, I forgot about the comment